examples/all_the_things.py

```diff
@@ -119,5 +119,6 @@ def teardown(test):
   #  with open('my_private_key.json', 'r') as json_file:
   #    test.AddOutputCallbacks(output.UploadToMfgInspector.from_json(
   #        json.load(json_file)))
-  test.Configure(teardown_function=teardown)
-  test.Execute(test_start=triggers.PromptForTestStart())
+  # test.Configure(teardown_function=teardown)
```

Comment:
hm, maybe don't change this file? don't think it matters much, but we should probably leave it in some kind of base-state in the repo and only change it locally for testing.